### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by SilverTee**
Translated:

death.js   - Deathmessage fixed (was useless until this push =] )
glove.js - - Change some letters.
hammer.js - fixed some sentences
quiver.js - fixed some sentences 
amulet.js - fixed one sentence 
armor.js - fixed some sentences 
bag.js - fixed some words
boot.js - fixed some words
bow.js - - fixed some words =P
glove.js - fixed some letters

### CODE REVIEW

File: loc/items/bow.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ export default [
       en: 'Put together with the bones of... somebody?',
       ru: 'Cобран из костей... кого-то?',
       fr: 'Assemblé avec les os de... quelqu\'un?',
-      de: 'Zusammen mit den Knochen von... jemandem?',
+      de: 'Zusammengebaut mit dem knochen von... jemandem?',
```
Comment (dekdevy) [inline loc/items/bow.js]: den Knochen*

File: loc/items/bow.js [hunk 2]
```diff
@@ -168,7 +168,7 @@ export default [
       en: 'An expert bow of an elegant race.',
       ru: 'Искусный лук изящный расы',
       fr: 'Un arc d\'expert venant d\'une race élégante.',
-      de: 'Ein erfahrener Bogen eines eleganten Rennens.',
+      de: 'Ein experten Boogen von einer intelligenten Rasse.',
```
Comment (dekdevy) [inline loc/items/bow.js]: Expertenbogen

File: loc/items/glove.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ export default [
     description: {
       en: 'Clicking sounds travel with you as you wear these.',
       ru: 'Клацающие звуки путешествуют с вами когда вы носите эти перчатки.',
-      de: 'Klackende Geräusche reisen mit, während Sie diese tragen.',
+      de: 'Klackende Geräusche begleiten sie, während ihrer Benutzung.',
```
Comment (dekdevy) [inline loc/items/glove.js]: Komma kann weg

File: loc/items/hammer.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ export default [
       ru: 'Это должны быть гвозди, а не черепа.',
       hi: 'इससे कील ठोकते है, खोपड़ी नहीं।',
       es: 'Esto debería estar clavando clavos, no cráneos.',
-      de: 'Das sollten hämmernde Nägel sein, nicht Schädel.',
+      de: 'Das sollten Nägel hämmern, nicht Schädel.',
```
Comment (dekdevy) [inline loc/items/hammer.js]: Der*? I think

**Comment by dekdevy** [comment]
Hey. Nice work, thank you a lot. I commented some stuff, if you change that then it'll be perfect.

File: loc/items/bow.js (hunk 1 above)
Comment (SilverTee) [reply loc/items/bow.js]: uff :D, werde ich anpassen danke!

**Pull request merged**